src/data/ConsoleContext.cpp

```diff
@@ -109,7 +109,7 @@ const std::vector<ConsoleContext::MemoryRegion> GameBoyConsoleContext::m_vMemory
     { 0x0000U, 0x00FFU, ConsoleContext::AddressType::HardwareController, "Interrupt vector" },
     { 0x0100U, 0x014FU, ConsoleContext::AddressType::ReadOnlyMemory, "Cartridge header" },
     { 0x0150U, 0x3FFFU, ConsoleContext::AddressType::ReadOnlyMemory, "Cartridge ROM (fixed)" }, // bank 0
-    { 0x4000U, 0x7FFFU, ConsoleContext::AddressType::ReadOnlyMemory, "Cartridge ROM (paged)" }, // bank 1-XX (switchable)
+    { 0x4000U, 0x7FFFU, ConsoleContext::AddressType::ReadOnlyMemory, "Cartridge ROM (paged)" }, // bank 1-7 for GBC (switchable)
```

Comment:
There can be a lot more than 7 ROM banks: http://gbdev.gg8.se/wiki/articles/Memory_Bank_Controllers#4000-7FFF_-_ROM_Bank_00-1FF_.28Read_Only.29

Reply:
Misread ROM as RAM, will revert.